src/Sylius/Component/ImportExport/spec/ExporterSpec.php

```diff
@@ -0,0 +1,216 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\ImportExport;
+
+use Doctrine\ORM\EntityManager;
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Psr\Log\LoggerInterface;
+use Sylius\Component\ImportExport\Logger\ImportExportLogger;
+use Sylius\Component\ImportExport\Model\ExportJobInterface;
+use Sylius\Component\ImportExport\Model\ExportProfileInterface;
+use Sylius\Component\ImportExport\Provider\CurrentDateProviderInterface;
+use Sylius\Component\ImportExport\Reader\ReaderInterface;
+use Sylius\Component\ImportExport\Writer\WriterInterface;
+use Sylius\Component\Registry\ServiceRegistryInterface;
+use Sylius\Component\Resource\Repository\RepositoryInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ * @author Łukasz Chruściel <lukasz.chrusciel@lakion.com>
+ */
+class ExporterSpec extends ObjectBehavior
+{
+    function let(
+        CurrentDateProviderInterface $dateProvider,
+        EntityManager $entityManager,
+        RepositoryInterface $exportJobRepository,
+        ServiceRegistryInterface $readerRegistry,
+        ServiceRegistryInterface $writerRegistry
+    )
```

Comment:
**PSR-2**: "When the argument list is split across multiple lines, the closing parenthesis and opening brace MUST be placed together on their own line with one space between them."